lib/api.js

```diff
@@ -32,6 +32,10 @@ api.declare({
   scopes:      [['secrets:set:<name>']],
   title:       'Set Secret',
   stability:    'stable',
+  cleanPayload: payload => {
```

Comment:
Hah, well, that would work if we were using ES6.  This service hasn't seen much love!